scss/_navbar.scss

```diff
@@ -203,6 +184,18 @@
         .navbar-toggler {
           display: none;
         }
+
+        .dropup {
+          .dropdown-menu {
+            position: absolute;
```

Comment:
`position` can be removed because it is already defined in line 157
https://github.com/twbs/bootstrap/blob/579add5809b8f4b6307da26cbc0cce02c4a9386d/scss/_navbar.scss#L157